spec/controllers/flood_risk_engine/enrollments/steps_controller_spec.rb

```diff
@@ -28,7 +28,7 @@ module FloodRiskEngine
         end
 
         describe "back step (step == (enrollment.step - 1)" do
-          let(:step) {steps[0]}
+          let(:step) { steps[0] }
```

Comment:
All of the changes in this file are of this type and look good to me.

On a general not - this was missed before because a lot of rules were skipped for specs - I think these changes suggests that was over done.